static/bidder-info/qt.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "https://endpoint1.qt.io/pserver"
+maintainer:
+  email: "octavian.cimpu@qt.io"
```

Comment:
@qt-io prebid team has sent email to verify above contact. Requesting to provide feedback on email thread

Reply:
This is not a valid maintainer email address. You must use a group email rather than a specific individual. Please see Global Module Rule 1 in our [Module Rules documentation](https://docs.prebid.org/dev-docs/module-rules.html#global-module-rules).